internal/app/replication/mysql.go

```diff
@@ -0,0 +1,151 @@
+package replication
+
+import (
+	"database/sql"
+
+	"github.com/yandex/mysync/internal/log"
+	"github.com/yandex/mysync/internal/mysql"
+)
+
+type ExternalReplication struct {
+	UnimplementedExternalReplication
+	IsExternalReplicationSupported bool
```

Comment:
This field should be removed. If we use ExternalReplication it means that setting is True